README.rst

```diff
@@ -65,11 +68,17 @@ To create the SageMaker MXNet Container Python package:
     cp dist/sagemaker_mxnet_container-<package_version>.tar.gz docker/<mxnet_version>/final
 
 Once you have those binaries, you can then build the image.
-The Dockerfiles expect two build arguments:
 
-- ``py_version``: the Python version.
+
+For MXNet versions 1.1.0-1.3.0, the Dockerfiles expect two build arguments:
```

Comment:
hmmm, do you think version specifics should exist in their respective branch/tag?

Reply:
that could clean the README up quite a bit. how about I do it in a following PR?